ds701_book/jupyter_notebooks/19-Regression-III-More-Linear.ipynb

```diff
@@ -7,39 +7,27 @@
         "---\n",
```

Comment:
<p>Suggest adding a sentence stating what is the y variable we are trying to predict.</p><p> </p><p>Also, should we define the columns just as an fyi?</p><p> </p><p>TOTEMP - Total Employment</p><p>        GNPDEFL - GNP deflator</p><p>        GNP - GNP</p><p>        UNEMP - Number of unemployed</p><p>        ARMED - Size of armed forces</p><p>        POP - Population</p><p>        YEAR - Year (1947 - 1962)</p><p> </p><p>From</p><p><a href="https://www.statsmodels.org/stable/datasets/generated/longley.html" target="_blank">https://www.statsmodels.org/stable/datasets/generated/longley.html</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tools4ds/DS701-Course-Notes/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I suggest printing y.head() too.</p>

Reply:
<p>Done. y is a series so doesn't have a column name.</p>